bin/crew

```diff
@@ -471,6 +471,26 @@ def help(pkgName)
   end
 end
 
+def cache_build
+  @build_cachefile = "#{CREW_CACHE_DIR}#{@pkg.name}-#{@pkg.version}-build-#{@device[:architecture]}.tar.zst"
+  if CREW_CACHE_ENABLED && File.writable?(CREW_CACHE_DIR)
```

Comment:
Should there be a message to indicate if `CREW_CACHE_DIR` is not writable?  Otherwise, nothing happens without any feedback?

Reply:
Sure.

Reply:
I'm not seeing a error message line.

Reply:
See 
```
  else
    puts 'CREW_CACHE_ENABLED is not set.'.lightred unless CREW_CACHE_ENABLED
    puts 'CREW_CACHE_DIR is not writable.'.lightred unless File.writable?(CREW_CACHE_DIR)
  end
```

Reply:
Oh, sorry, missed it.  Is `CREW_CACHE_ENABLED is not set.` an error?  Maybe that should be a warning instead?  I'm a never enable cache if not needed kinda guy myself.